data/map.txt

```diff
@@ -20073,6 +22976,68 @@ system Polerius
 			distance 403
 			period 28.7798
 
+system Poli'kor
+	pos -26.888889 116.33333
+	government Korath
+	attributes korath warm
+	asteroids "small rock" 2 2.079
+	asteroids "medium rock" 18 2.5872
+	asteroids "large rock" 7 1.6632
+	asteroids "small metal" 38 2.6334
+	asteroids "medium metal" 155 2.2407
+	asteroids "large metal" 5 1.9173
+	minables iron 48 2.87932
+	minables titanium 16 2.87553
+	minables uranium 13 3.05224
+	haze _menu/haze-orange
+	fleet "Korath Raid" 1600
+	habitable 2560
+	belt 1444
+	trade Clothing 312
+	trade Electronics 738
+	trade Equipment 427
+	trade Food 565
+	trade "Heavy Metals" 1253
+	trade Industrial 822
+	trade "Luxury Goods" 1164
+	trade Medical 471
+	trade Metal 465
+	trade Plastic 406
+	hazard "Core Sheltered Heat" 16
+	object
+		sprite star/giant
```

Comment:
The old giant is kept only for backwards compatibility, it's intended to be removed in the future.  Probably not a good idea to add them.

Can't say much better for the `-old` stars either, I'd rather rename `k5-old` to something like `m-subgiant` or `m5iv`, but that's further down the road if ever.

```suggestion
		sprite star/k5-old
```